create-doks-with-terraform-flux/README.md

```diff
@@ -1 +1,470 @@
+# Gitops stack using DOKS and Flux CD
 
+This blueprint will guide you step by step on how to spin up a single DOKS (DigitalOcean Kubernetes) cluster and Flux CD for managing application deployments in a GitOps fashion. In the end we will also tell [Flux CD](https://fluxcd.io) to perform a basic deployment of the now ubiquitous `busybox` Docker application.
```

Comment:
```suggestion
This blueprint will guide you on how to use [Flux](https://fluxcd.io) to manage application deployments on a DigitalOcean Kubernetes (DOKS) cluster in a GitOps fashion. In the end, we will also perform a basic deployment of the `busybox` Docker application.
```

Reply:
Thanks for the suggestion. Fixed in the latest commit.